multiversx_sdk/abi/fields.py

```diff
@@ -0,0 +1,26 @@
+import io
+from typing import Any, Dict, List
+
+from multiversx_sdk.abi.interface import SingleValue
+
+
+class Field:
+    def __init__(self, name: str, value: SingleValue) -> None:
+        self.name = name
+        self.value = value
+def encode_fields_nested(fields: List[Field], writer: io.BytesIO):
```

Comment:
add empty lines between class and function

Reply:
Fixed in https://github.com/multiversx/mx-sdk-py/pull/49.